creator/lib/screens/login_screen.dart

```diff
@@ -8,10 +17,221 @@ class LoginScreen extends StatefulWidget {
 }
 
 class _LoginScreenState extends State<LoginScreen> {
+  bool _loginInProgress = false;
+
+  // ログインボタン
+  Widget _buildLoginButtons(BuildContext context) {
+    return Container(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildLoginButton(
+            context,
+            'assets/signin_button/google_logo.png',
+            'Googleでログイン',
+            () {
+              _tryLoginWith(
+                context,
+                signInWithGoogle(),
+              );
+            },
+          ),
+          SizedBox(
+            height: 16,
+          ),
+          _buildLoginButton(
+            context,
+            'assets/signin_button/apple_logo.png',
+            'Appleでログイン',
+            () {
+              _tryLoginWith(
+                context,
+                signInWithApple(),
+              );
+            },
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildLoginButton(
+    BuildContext context,
+    String asset,
+    String text,
+    Function() method,
+  ) {
+    final mq = MediaQuery.of(context).size;
+    final screenWidth = mq.width;
+    final screenHeight = mq.height;
+
+    return Center(
+      child: Container(
+        width: screenWidth < 600 ? screenWidth * 0.8 : screenWidth * 0.5,
+        height: screenHeight * 0.06,
+        child: ElevatedButton(
+          style: Theme.of(context).elevatedButtonTheme.style?.copyWith(
+                backgroundColor: MaterialStateProperty.all(Colors.white),
+                foregroundColor: MaterialStateProperty.all(Colors.black),
+              ),
+          onPressed: method,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              _buildLoginMethodLogo(asset),
+              SizedBox(
+                width: 16,
+              ),
+              Text(
+                text,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Container _buildLoginMethodLogo(String asset) {
```

Comment:
Please make sure all functions return Widget (base class)

Reply:
I checked all function of Widget to "Widget".